libs/ui/src/display_settings/sensory_toggle_settings.tsx

```diff
@@ -0,0 +1,33 @@
+import { ThemeConsumer } from 'styled-components';
+import React from 'react';
+import { SettingsPane } from './settings_pane';
+import { Button } from '../button';
+import { DisplaySettingsManagerContext } from '../display_settings_manager_context';
+import { appStrings } from '../ui_strings';
+
+export interface SensoryToggleSettingsProps {}
+
+export function SensoryToggleSettings(): JSX.Element {
```

Comment:
I called this "SensoryToggle" because we expect to add an option for visual-only mode here